app/assets/javascripts/darkswarm/controllers/products_controller.js.coffee

```diff
@@ -50,7 +50,7 @@ Darkswarm.controller "ProductsCtrl", ($scope, $filter, $rootScope, Products, Ord
   $scope.appliedPropertiesList = ->
     $scope.activeProperties.map( (property_id) ->
       Properties.properties_by_id[property_id].name
-    ).join(" & ") if $scope.activeProperties?
+    ).join(" " + t('shop.products.filter_properties_or' ) + " ") if $scope.activeProperties?
```

Comment:
In Javascript it's better to use our Angular filter for translations:
```
join(" {{ 'shop.products.filter_properties_or' | t }} ")
```
In some cases we had problems with the global `t` function not working when called directly. I think it depends on the scope in which the code is running. Using the `t` filter is the safer option and we should try to be consistently using only that one in Javascript.

We should probably document that somewhere. Which documents did you read to become familiar with this?

Reply:
@mkllnk - ah, I was not aware of that so thanks.  I will change to use the angular filter instead.

Reply:
@sauloperez  @mkllnk  - Added doc's to suggested page: (https://github.com/openfoodfoundation/openfoodnetwork/wiki/i18n#javascript-translations-prefer-angular-filter-over-global-t-function)